### PR TITLE
dynamic sizing

### PR DESCRIPTION
**Pull request by arnaudlewis**
add viewBox property for dynamic sizing and wigth/height set to 100% for the component

### CODE REVIEW

File: src/MultiSlider.js [hunk 1]
```diff
@@ -47,20 +47,22 @@ var MultiSlider = React.createClass({
...
+   if (useTouches()) {
       // There is a bug in touch events and we need to compute the real clientX
       // http://stackoverflow.com/questions/5885808/includes-touch-events-clientx-y-scrolling-or-not
-      clientX = e.pageX; // safer to start with pageX
+      var clientX = e.pageX; // safer to start with pageX
       var stopAt = document.body.parentElement;
       while (node && node !== stopAt) {
         clientX -= node.scrollLeft;
-        node = node.parentElement;
+        p.x = clientX;
```
Comment (gre) [inline src/MultiSlider.js]: why this change? it looks like this will break on mobile.
I think `xForEvent` function should not be changed.

Comment (gre) [inline src/MultiSlider.js]: Ok. I think I get it now.
I guess the previously returning `clientX-bound.left` behavior MUST be preserved (try to scroll right on the example pages and see what will happen now ;) ) but that you can just apply the final matrixTransform at the end.

Are we sure this works everywhere? (what does caniuse say?)

Comment (arnaudlewis) [inline src/MultiSlider.js]: j’ai apporté la modif.
Ca marche nickel sur ie9 et sur mobile aucun souci ;)

**Comment by gre** [comment]
There is 2 times the xForEvent method in this PR :-D

**Comment by gre** [comment]
thanks

**Comment by gre** [comment]
released `v2.0.0`
